test/helper.js

```diff
@@ -87,7 +97,65 @@ mocha.before(async () => {
     await helper.Roles.setup();
   }
 
+  //overwrites.publisher = await Exchanges.connect({fake:true});
+
   if (!helper.hasPulseCredentials()) {
+    helper.Auth = class {
+      ping() {
+        return Promise.resolve();
+      }
+      client(){
+        return Promise.resolve();
+      }
+      createClient(){
+        helper.events.triggerEvents(helper.authEvents, clientCreated());
+        return Promise.resolve();
+      }
+      deleteClient(){
+        return Promise.resolve();
+      }
+    };
+
+    helper.auth = new helper.Auth();
+
+    helper.authEvents ={
+      clientCreated: () => 'client created',
+      clientDeleted: () => 'client deleted',
+    };
+
+    class EventHandler {
```

Comment:
This appears to be a fake version of [PulseTestReceiver](https://www.npmjs.com/package/taskcluster-lib-testing#pulsetestreceiver) which is already a testing utility and doesn't need to be faked :)